src/eclipse/org.eclipse.jdt.ui.prefs

```diff
@@ -55,8 +55,8 @@ eclipse.preferences.version=1
 formatter_profile=_Spring
 formatter_settings_version=12
 org.eclipse.jdt.ui.ignorelowercasenames=true
-org.eclipse.jdt.ui.importorder=java;javax;org;com;\#;
+org.eclipse.jdt.ui.importorder=java;javax;;org.springframework;\#;
```

Comment:
I think it supports more than the first part. I removed `com` because it doesn't match the import order [described on the wiki](https://github.com/spring-projects/spring-framework/wiki/Code-Style#import-statements).

Reply:
Indeed it does support `org.springframework`.

That's a new feature. At least the last few times I checked, that didn't work. So... good to know that that's now supported! 👍

Reply:
> I removed `com` because it doesn't match the import order described on the wiki.

Ahhh... then the wiki is wrong.

I created/modified the Eclipse import order ages ago based on a one-on-one discussion with @jhoeller.

We actually import `com.*` before `org.*` and both of those before `org.springframework.*`.

Reply:
I fixed that in c14f83e7276d96ca103ae30a2aa55bc7edfc95f2.

Reply:
I'll update the wiki as well.

Reply:
done!

Reply:
@sbrannen Are you sure that's still the case? I understand that quite a few discussions were had when the wiki was created and that it's used as the basis for the [IntelliJ settings](https://github.com/spring-projects/spring-framework/blob/master/src/idea/spring-framework.xml#L17-L29). If that is the preferred order, I'll need to also update checkstyle.

Reply:
I do not differentiate between particular prefixes there: It's simply "all other imports" in that section, with "com." and "org." being common suspects but not specifically ordered within that section. Even if the alphabetical ordering usually leads to "com." immediately followed by "org.", there may be some other imports (e.g. "kotlin.") inbetween; look at BeanUtils for an example. Let's reflect that in the settings as well as on the wiki please.

Reply:
Thanks. I've reverted the commits on master and the wiki since I think that's what we had.

Reply:
> I do not differentiate between particular prefixes there

Well, that's not what you told me back in (I believe) 2007, but that's fine. Times change. 😉 

> Let's reflect that in the settings as well as on the wiki please.

Sure. I'll address that pronto.

Reply:
Oh, wait... @philwebb already reverted the change in `master` as I was working on it.

Reply:
And... he reverted the change in the wiki. 

So... nothing to see here. Carry on...

Reply:
> I've reverted the commits on master and the wiki since I think that's what we had.

Aha... just now seeing this.

The Internet connection where I'm on vacation is really _suboptimal_. 😞 

C'est la vie...